docker/Dockerfile.cpu

```diff
@@ -4,60 +4,78 @@
 # architectures using GCC compilers and OpenMPI.
 ##############################################################
 
-ARG OMPI_BRANCH="v4.1.4"
-
 # Base image 
-FROM ubuntu:22.04 as base
+FROM ubuntu:22.04 AS base
+
+ARG gcc=""
+
+ENV DEBIAN_FRONTEND=noninteractive
 
-ENV DEBIAN_FRONTEND noninteractive
+# Add repo for other python versions
+RUN apt-get update && \
+    apt-get install -y software-properties-common
 
 # Install python
 RUN apt-get update && \
-    apt-get install -y dh-autoreconf python3-venv python3-dev python3-pip
+    apt-get install -y dh-autoreconf python3-venv python3-dev python3-pip gdb
```

Comment:
as written above, maybe we do this in Dockerfile.devito instead?

Reply:
No it's a pain, the `.devito` is only adding devito to the base, and installing things in it defeats the purpose of multistage and minimal final image